scqc_nf.sh

```diff
@@ -1,76 +1,85 @@
-// before running:'source scqc_reqs-0.1'
-// standard run case:'nextflow run some_path/scqc_nf.sh -c config_file &'
+// before running: 'source scqc_reqs-1.0'
+// standard run case:'nextflow run scqc_nf.sh -c PIP-3144.config -with-dag flowchart.png -with-report -resume'
 
-//human_index='/ei/cb/common/Scripts/scqc/References/human_w_ercc.idx'
-//mouse_index='/ei/cb/common/Scripts/scqc/References/mm_ERCC.idx'
-//chicken_index='/ei/cb/common/Scripts/scqc/References/chicken_ERCC.idx'
+nextflow.enable.dsl=2
 
-image_scater='/ei/cb/common/Scripts/scqc/Containers/R-3.5.2_scater.img'
-image_json='/ei/cb/common/Scripts/scqc/Containers/R-3.5.2_bioMjsonS3.img'
-image_doc='/ei/cb/common/Scripts/scqc/Containers/verse.simg'
+image_scater='--overlay /ei/cb/common/Scripts/singlecellQC/Containers/R-scater.v7.uwat_overlay.img:ro /ei/cb/common/Scripts/singlecellQC/Containers/R_scater_1.22.img'
+image_json='/ei/cb/common/Scripts/singlecellQC/Containers/R-3.5.2_bioMjsonS3.img'
+image_doc='/ei/cb/common/Scripts/singlecellQC/Containers/R_verse.dplyr_1.0.8.img'
+image_gs='/ei/cb/common/Scripts/singlecellQC/Containers/ghost_script.img'
+image_kallisto='/ei/cb/common/Scripts/singlecellQC/Containers/kallisto_0.44.0.img'
 
-qc_script='/ei/cb/common/Scripts/scqc/Scripts/scqc_from_matrix.R'
-merge_script='/ei/cb/common/Scripts/scqc/Scripts/merge_kallisto_quant.R'
-k_scrape_script='/ei/cb/common/Scripts/scqc/Scripts/kallisto_mapping_scrape.R'
-doc_script='/ei/cb/common/Scripts/scqc/Scripts/QCreport.Rmd'
-plate_matrix_merge_script='/ei/cb/common/Scripts/scqc/Scripts/plate_merge.R'
-tx2g_script='/ei/cb/common/Scripts/scqc/Scripts/est_counts_tx2gene.R'
+qc_script='/ei/cb/development/lany/git/singlecellQC/main_dev_branch/Scripts/scqc_from_matrix.meta.R' 
+merge_script='/ei/cb/development/lany/git/singlecellQC/main_dev_branch/Scripts/merge_kallisto_quant.R'
+k_scrape_script='/ei/cb/development/lany/git/singlecellQC/main_dev_branch/Scripts/kallisto_mapping_scrape.R'
+doc_script='/ei/cb/development/lany/git/singlecellQC/main_dev_branch/Scripts/QCreport.Rmd'
+plate_matrix_merge_script='/ei/cb/development/lany/git/singlecellQC/main_dev_branch/Scripts/plate_merge.R'
+tx2g_script='/ei/cb/development/lany/git/singlecellQC/main_dev_branch/Scripts/est_counts_tx2gene.R'
```

Comment:
lines 12-17, the path should be changed from 
`/ei/cb/development/lany/git/singlecellQC/main_dev_branch`
to 
`/ei/cb/common/Scripts/scqc`
OR 
`/ei/software/cb/singlecellQC/1.0/x86_64/bin`.

we should not use `/ei/cb/development` for hosting production scripts as this location will be cleaned up and files will get removed.

Reply:
scripts are now under /ei/software/cb/singlecellQC/1.0/x86_64/bin/Scripts